sn_networking/src/lib.rs

```diff
@@ -87,6 +95,8 @@ pub const fn close_group_majority() -> usize {
 
 /// Max duration for all GET attempts
 const MAX_GET_RETRY_DURATION_MS: u64 = 6800;
+
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
wasm doesn't need `MAX_GET_RETRY_DURATION` ?
I thought this is a non-transport-dependent retry config ?

Reply:
we cannot (at the moment) easily have backoff with wasm32. backoff uses tokio right now.

We can probably rejig this, but i'd say that's an optimisation atm. The main aim here (i think) is to get the wasm code building and checked?